### PR TITLE
feat: replace `unruggable-core` with `unruggable-sdk`

### PR DESCRIPTION
**Pull request by remiroyc**
## Description
This PR proposes to replace the current implementation using `unruggable-core` with the newly published `unruggable-sdk` package. The SDK provides a more developer-friendly experience with:

- Improved TypeScript support
- More concise and readable API
- Maintained wrapper around the core package functionality

## Changes
- Replace `unruggable-core` dependency with `unruggable-sdk`
- Update relevant implementations to use the new SDK interface
- Update types where necessary

## Benefits
- Better developer experience through improved typing
- More maintainable code through cleaner abstractions
- Same core functionality with a more ergonomic interface

### CODE REVIEW

**Comment by ponderingdemocritus** [comment]
looks good - fix conflicts then lets ship

File: packages/plugin-starknet/src/actions/unruggable.ts [hunk 1]
```diff
@@ -127,77 +132,62 @@ export const deployToken: Action = {
...
+                        ).toFixed(0),
+                    },
+                    {
+                        address: ACCOUNTS.BLOBERT,
+                        amount: new Percent(
+                            2.5,
+                            response.initialSupply
+                        ).toFixed(0),
+                    },
+                ],
```
Comment (enitrat) [inline packages/plugin-starknet/src/actions/unruggable.ts:182]: eliza + blobert ? eliza has a starknet wallet as well ?

File: packages/plugin-starknet/src/actions/unruggable.ts [hunk 2]
```diff
@@ -127,77 +132,62 @@ export const deployToken: Action = {
...
-            const tx = await account.execute([...deployCalls, ...launchCalls]);
+
+            await launchOnEkubo(config, {
+                antiBotPeriodInSecs: 3600,
+                currencyAddress: TOKENS.LORDS,
+                fees: "3",
+                holdLimit: "2",
+                memecoinAddress: tokenAddress,
+                starknetAccount: account,
+                startingMarketCap: "5000",
```
[the remark this answers is not in the document]
Comment (enitrat) [reply packages/plugin-starknet/src/actions/unruggable.ts:166]: why?

We can cap from 5k to like 30k

[the remark this answers is not in the document]
Comment (enitrat) [reply packages/plugin-starknet/src/actions/unruggable.ts:166]: 10k by default would already be better imo

**Review by lalalune — DISMISSED** [review]

File: packages/plugin-starknet/src/actions/unruggable.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (odilitime) [reply packages/plugin-starknet/src/actions/unruggable.ts:171]: we can tweak after the merge if it's a problem

**Review by odilitime — DISMISSED** [review]

**Review by odilitime — APPROVED** [review]

**Pull request merged**